src/main/java/net/sf/jabref/logic/exporter/FileSaveSession.java

```diff
@@ -90,14 +84,7 @@ public void commit(Path file) throws SaveException {
                 LOGGER.error("Error when creating lock file.", ex);
             }
 
-            FileUtil.copyFile(temporaryFile.toFile(), file.toFile(), true);
-        } catch (IOException ex2) {
-            // If something happens here, what can we do to correct the problem? The file is corrupted, but we still
-            // have a clean copy in tmp. However, we just failed to copy tmp to file, so it's not likely that
-            // repeating the action will have a different result.
-            // On the other hand, our temporary file should still be clean, and won't be deleted.
-            throw new SaveException("Save failed while committing changes: " + ex2.getMessage(),
-                    Localization.lang("Save failed while committing changes: %0", ex2.getMessage()));
```

Comment:
Please remove this string from the localization files. This should fix the failing travis build. Otherwise LGTM